.github/workflows/release-sprint.yml

```diff
@@ -24,27 +26,21 @@ jobs:
 
     strategy:
       matrix:
-        branch_tag: ['master:v1.7.0']
+        branch_tag: ['master:v1.8.0', 'v1.7.x:v1.7.1', 'v1.6.x:v1.6.3']
 
     steps:
-    - name: Setup Git
+    - name: Install dependencies
       run: |
-        gh auth setup-git
-
-    - uses: anchore/sbom-action/download-syft@v0.17.2
-
-    - run: |
-        git config --global user.email "dko@suse.com"
-        git config --global user.name "David Ko"
+        python -m pip install requests
 
-    - uses: actions/checkout@v4
+    - name: Checkout repository
+      uses: actions/checkout@v3
```

Comment:
Why changed version to v3 from v4?

Reply:
Oh, it is copied from other repo's workflow.
Updated it to v4, thank you.